### DISCUSSION TITLE
Support Package Provisioning for All Users

### DISCUSSION THREAD

**Discussion by shockogit** [microsoft/winget-cli#5101] [Q&A]
MS Store is disabled in our environment and as such we are adding the AppInstaller package _**offline**_ using the [add-appxpackage](https://learn.microsoft.com/en-us/powershell/module/appx/add-appxpackage?view=windowsserver2025-ps) CMDLET. This installs winget **_per user_** but we'd like to provision it into windows so it's available for all users. We cannot use the [Add-AppXProvisionsedPackage](https://learn.microsoft.com/en-us/powershell/module/dism/add-appxprovisionedpackage?view=windowsserver2025-ps) as the formats that the Winget offline bundle comes in do not support this CMDLET. 

Q: Is there existing support for system wide provisioning and if not, why not? It would seem to make sense to have this capability.

**Comment by florelis** [comment]
> We cannot use the [Add-AppXProvisionsedPackage](https://learn.microsoft.com/en-us/powershell/module/dism/add-appxprovisionedpackage?view=windowsserver2025-ps) as the formats that the Winget offline bundle comes in do not support this CMDLET.

What do you mean? That cmdlet should definitely work with the .msixbundle

You can also use `Repair-WinGetPackageManager -AllUsers` from the `Microsoft.WinGet.Client` module, although that would fetch the bundle instead of using an existing offline copy.
